### DISCUSSION TITLE
What do 🌻, dahlias, hibiscus, 🌷and calla lilies have in common?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1124] [Galaxy Trivia] [answered]
**Bonus:** How many rings do catta lemurs have on their tails?

**Comment by LeleDallas** [comment]
They are all commonly recognized as ornamental plants that are cultivated for their beauty and aesthetic appeal.

**Comment by LeleDallas** [comment]
> **Bonus:** How many rings do catta lemurs have on their tails?

Google said 13 😅

**Comment by peterbaccaro** [comment, marked as answer]
Dahlias, hibiscus and calla lilies have a number of things in common:
- They are **Flowering Plants**.
- They are **Broad Varieties** as there is a large variety of cultivars and species within each of these plant groups.
- They are **Popular Ornamental Plants** for ornamental gardening and landscaping.
- They are typically grown as **Perennials** in suitable climates meaning that they come back and bloom year after year.
- They grow from **Bulb or Tuberous Roots**.
- They are **Used in Flower Arrangements** due to their beauty.

Ring-tailed lemurs (Lemur catta) are named for the **13 alternating black and white bands** that adorn their tails.

**Reply by ParanoidUser** [reply]
All of them are odorless 👃🌻
